hax/minitrees.py

```diff
@@ -326,6 +351,10 @@ def load_single_dataset(run_id, treemakers, preselection=None, force_reload=Fals
     for i in range(1, len(dataframes)):
         result = _merge_minitrees(result, dataframes[i])
 
+    #
+    if is_blind(run_id):
```

Comment:
This will error if you're not loading basics, since the variables in the blinding cut are not defined